project2/index.html

```diff
@@ -6,28 +6,56 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:700,900" rel="stylesheet">
     <title>GD2Project2SP19</title>
 </head>
 
 <body>
-    <section.class-background>
-        <img src="IrisBook.jpg" alt="BookCover">
+
+    <section class="background">
```

Comment:
You created a lot of layout problems for yourself because your markup is organized so different than your design -- which forced you to position a lot of things. Can you explain why you kept things in this order (ie, the image first vs the blockquote)?

[Here's a CodePen](https://codepen.io/angeliquejw/pen/9724eef47cbd7976b0910419c8cd4847?editors=0100) that shows how this gets a lot simpler when the markup and design are more in line.